src/guide/keys.jl

```diff
@@ -40,7 +40,7 @@ function Guide.render(guide::Guide.ShapeKey, theme::Gadfly.Theme, aes::Gadfly.Ae
         colors = collect(keys(aes.color_key_colors))
     end
     
-    title_context, title_width = Guide.render_key_title(guide_title, theme)
```

Comment:
is `render_key_title` to be replaced by `render_key_title2` in gadfly 2.0?  if so, might add a comment in the code about that somewhere.  and create an issue to which we can add the 2.0 milestone tag.

Reply:
`render_key_title2` goes with `render_discrete_key`, whereas `render_key_title` is used by  `render(guide::ColorKey, ...)`.  I plan to make all the _discrete_ asesthetics use `render_discrete_key`, after which only the _continuous_ color key will be using the current `render_key_title`.  I don't have any plans for changing the continuous color key at the moment.